migrations/20210602_introductions.sql

```diff
@@ -0,0 +1,26 @@
+CREATE TABLE IF NOT EXISTS conn_intros (
```

Comment:
now that we have migrations in place why do we need `IF NOT EXISTS`? it should not exist - if it exists at this point (when this migration hasn't been played out and registered in database) then the database is in potentially in inconsistent state / compromised / anything else - just fail with error and let the user figure out why he has this inconsistency in the first place

Reply:
shouldn't we at least also have `drop if exists` in this case then? By only having `create if not exists` we basically leave it to chance that if the table does exist it is coincidentally consistent with the rest of the schema and application - this leads to non idempotent migrations and opens up a way for cryptic errors in application. I just don't see any advantages to having only `create if not exists` except for possible minor conveniences during development

Reply:
I either do not see any case in which `if not exists` may be needed